data/translations/spanish/meta.toml

```diff
@@ -0,0 +1,2 @@
+title = "Inscríbase para votar"
+description = "Vea las guías de votación en su estado"
```

Comment:
We're going to need a `title = "*"` and `description = "*"` `toml` file [here as well](https://github.com/18F/vote-gov/blob/bd24098e298bcc14604ee6c9b8d6e99538546c91/data/translations/english/meta.toml) @jpyuda as the English site will have those sections blank if they aren't referenced in the code.

🔗 https://github.com/18F/vote-gov/blob/bd24098e298bcc14604ee6c9b8d6e99538546c91/data/translations/english/meta.toml